benchmarks/zap_bench_test.go

```diff
@@ -86,7 +86,12 @@ func BenchmarkZapDisabledLevelsWithoutFields(b *testing.B) {
 
 func BenchmarkZapDisabledLevelsAccumulatedContext(b *testing.B) {
 	context := fakeFields()
-	logger := zap.NewJSON(zap.ErrorLevel, zap.Output(zap.Discard), zap.Fields(context...))
+	logger := zap.NewLogger(
```

Comment:
Optional: might be worth adding a helper to create a new JSON logger with output discarded, that takes a list of additional options on top.